_data/pt-BR.yml

```
@@ -0,0 +1,294 @@
+  liGithub: "@bisq-network no GitHub"
+  liTwitter: "@bisq-network no Twitter"
+  liYoutube: Canal da Bisq no YouTube
+  liSlack: Workspace da Bisq no Slack
+  liForum: Fórum Bisq
+  liMailingList: Lista de discussão de Contribuidores da Bisq
+  liFreenodeIRC: "#bisq no IRC da Freenode"
+  liSubreddit: Subreddit r/Bisq
+  liKeybase: "Equipe Keybase Bisq"
+
+main_nav:
+  liDownloads: Downloads
+  liLearnMore: Saiba Mais
+  liDocumentation: Documentação
+  liBlog: Blog
+  liCommunity: Comunidade
+  buttonDownloads: Downloads
+
+os_selector_options:
+  liAllDownloads: Todos os Downloads
+  liAllDownloadsText: Ir para a página de downloads
+  liWindows: Baixar para windows
+  liMac: Baixar para Mac
+  liDebian: Baixar para Debian/ubuntu
+  liRedHat: Baixar para Red Hat/Fedora
+  liArch: Baixar para Arch Linux
+  liArchText: "Repositório dos Usurários de Arch "
+  liSourceCode: Código Fonte
+  liReleaseNotes: Notas de Lançamento
+  liVerification: Verificação
+  liVerificationText1: Assinaturas PGP
+  liVerificationText2: Chave Pública PGP
+
+dao:
+  hLearnMore: "Quer saber mais?"
+  pLearnMore: "Você pode aprender mais sobre a DAO Bisq a partir dos recursos abaixo, desde documentos e vídeos de alto nível no estilo ELI5 até a especificação técnica para desenvolvedores."
+  conceptualOverview: Documento Conceitual com a Visão Geral
+  pConceptualOverview: "<p class=\"bigger\">Descrição <a href=\"https://docs.bisq.network/user-dao-intro.html\" target=\"_blank\" rel=\"noopener\">da DAO</a> em linguagem simples.</p>"
+  bisqDaoBrief: A DAO Bisq em pequenos vídeos
+  pBisqDaoBrief: "<p class=\"bigger\">Série de vídeos <a href=\"https://www.youtube.com/playlist?list=PLFH5SztL5cYPAXWFz-IMB4dBZ0MEZEG_e\" target=\"_blank\">curtos e rápidos</a> sobre a DAO Bisq.</p>"
+  technicalOverview: Visão Geral Técnica - Doc
+  pTechnicalOverview: "<p class=\"bigger\"><a href=\"https://docs.bisq.network/dao/specification.html\" target=\"_blank\" rel=\"noopener\">Especificação</a> do funcionamento da DAO Bisq e do token BSQ."
+  notConvinced: "Não está convencido de que um token é necessário?"
+  pNotConvinced: "<p class=\"bigger\">Nós temos um <a href=\"https://bisq.network/blog/bisq-dao-for-bitcoin-maximalists/\" target=\"_blank\">post</a> só para você.</p>"
+  seeMore: >
+    <p>Veja mais recursos em nosso<a href='https://docs.bisq.network/dao.html'> Tudo Sobre a DAO Bisq</a> doc.</p><p>Contribuidores anteriores, verifiquem<a href='https://docs.bisq.network/compensation.html#get-added-to-genesis'>o link</a>para obter instruções sobre como ser adicionado à distribuição de BSQ da gênese.</p>
```

```suggestion
    <p>Veja mais recursos em nosso<a href='https://docs.bisq.network/dao.html'> Tudo Sobre a DAO Bisq</a> doc.</p><p>Contribuidores anteriores, verifiquem <a href='https://docs.bisq.network/compensation.html#get-added-to-genesis'>o link</a> para obter instruções sobre como ser adicionado à distribuição de BSQ da gênese.</p>
```

Missing spaces around link.